src/test/regression/regression.test.ts

```diff
@@ -97,6 +97,10 @@ function replaceImports(data: string) {
       `'${require.resolve(
         "../../../node_modules/@types/react/index.d"
       )}'`.replace(/.d.ts/, "")
+    )
+    .replace(
+      /v2\/core\/util\/flowCompat/,
+      require.resolve("../../../src/flowCompat.ts").replace(/.ts/, "")
```

Comment:
> // The in memory typescript compiler can't seem to find imported modules.
// This hack to replace imports with absolute paths seems to work.
// Don't apply this before the snapshot, or your snapshot will have absoulte paths

source: existing code comment in lines above this diff